ray-on-gke/user/jupyterhub/jupyter_config/config.yaml

```diff
@@ -0,0 +1,78 @@
+# This file can update the JupyterHub Helm chart's default configuration values.
+#
+# For reference see the configuration reference and default values, but make
+# sure to refer to the Helm chart version of interest to you!
+#
+# Introduction to YAML:     https://www.youtube.com/watch?v=cdLNKUoMc6c
+# Chart config reference:   https://zero-to-jupyterhub.readthedocs.io/en/stable/resources/reference.html
+# Chart default values:     https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/HEAD/jupyterhub/values.yaml
+# Available chart versions: https://jupyterhub.github.io/helm-chart/
+hub:
+  config:
+    Authenticator:
+      admin_users:
+        - user1
+        - user2
+      allowed_users:
+        - user3
+        - user4
+    # ...
+    DummyAuthenticator:
+      password: password
+    JupyterHub:
+      authenticator_class: dummy
+  networkPolicy:
+    enabled: false
+
+prePuller:
+  hook:
+    enabled: false
+
+proxy:
+  chp:
+    networkPolicy:
+      enabled: false
+  traefik:
+    networkPolicy:
+      enabled: false
+
+scheduling:
+  userScheduler:
+    enabled: false
+
+singleuser:
+  nodeSelector:
+    iam.gke.io/gke-metadata-server-enabled: "true"
+  image:
+    name: jupyter/datascience-notebook
+    tag: python-3.10
+  startTimeout: 700
+  profileList:
+    - display_name: "Prof1"
```

Comment:
What are we trying to achieve with these profiles?
Maybe we could have 1 CPU, 1 GPU, 1 TPU example?
And also illustrate how to set memory limits?

Reply:
How do you request for a TPU? currently there is a request/limit field set in the profiles to get the gpus.

Reply:
TPUs are not supported yet (pending @yiyinglovecoding 's PR)